landlab/components/uniform_precip/generate_uniform_precip.py

```diff
@@ -13,7 +13,8 @@
 
 import random
 import numpy as np
-from landlab import Component
+from landlab import Component, ModelGrid
+from six import next
 
 
```

Comment:
Can you point out an example of a component already using this kind of notation? I've not seem one that doesn't commit to, e.g., `m` as the length scale (even though it might not have to be).